src/Maestro/Maestro.Web.Tests/SubscriptionsController20200220Tests.cs

```diff
@@ -0,0 +1,485 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Common;
+using Maestro.Data;
+using Maestro.Web.Api.v2020_02_20.Controllers;
+using Maestro.Web.Api.v2020_02_20.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.Internal.Testing.Utility;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.Internal;
+using Microsoft.Extensions.Internal;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+
+namespace Maestro.Web.Tests
+{
+    [TestFixture]
+    public class SubscriptionsController20200220Tests
+    {
+        const string testChannelName = "test-channel-sub-controller20200220";
+        const string defaultGitHubSourceRepo = "https://github.com/dotnet/sub-controller-test-source-repo";
+        const string defaultGitHubTargetRepo = "https://github.com/dotnet/sub-controller-test-target-repo";
+        const string defaultAzdoSourceRepo = "https://dev.azure.com/dnceng/internal/_git/sub-controller-test-source-repo";
+        const string defaultAzdoTargetRepo = "https://dev.azure.com/dnceng/internal/_git/sub-controller-test-target-repo";
+        const string deleteScenarioSourceRepo = "https://github.com/dotnet/sub-controller-delete-sub-source-repo";
+        const string deleteScenarioTargetRepo = "https://github.com/dotnet/sub-controller-delete-sub-target-repo";
+        const string triggerScenarioSourceRepo = "https://github.com/dotnet/sub-controller-trigger-sub-source-repo";
+        const string triggerScenarioTargetRepo = "https://github.com/dotnet/sub-controller-trigger-sub-target-repo";
+        const string defaultBranchName = "main";
+        const string defaultClassification = "classy-classification";
+        const uint defaultInstallationId = 1234;
+        TestData data;
```

Comment:
In general, fields are dangerous for tests, because it means they all share the same testdata/controller, which can lead to really weird behavioral interaction between tests.

In general, it's better for this to be a local variable.

If it really needs to be a field, it should be initialized in a [SetUp] method (that's how NUnit handles initialization/setup)

Reply:
done